flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/catalog/CatalogManager.java

```diff
@@ -316,6 +316,35 @@ public void dropTable(ObjectIdentifier objectIdentifier, boolean ignoreIfNotExis
 			"DropTable");
 	}
 
+	/**
+	 * Create a function in a given fully qualified path.
+	 *
+	 * @param catalogFunction  The function to put in the given path
+	 * @param objectIdentifier
+	 * @param ignoreIfExists
```

Comment:
It's better to add comments to explain arguments meaning.

Reply:
Done.